contents/blog/posthog-altinity-announce.md

```diff
@@ -0,0 +1,30 @@
+---
```

Comment:
I think we should also clearly link to our documentation on actually deploying using altinity cloud here - or do you think the link in marketplace is sufficient?

Reply:
That's the correct one :)